src/css/DarkMode.css

```diff
@@ -0,0 +1,81 @@
+:root {
```

Comment:
We have to use scss files instead of css, also we decided to use with BEM, look this http://getbem.com/introduction/